editor/plugins/plugin_config_dialog.cpp

```diff
@@ -305,7 +305,7 @@ PluginConfigDialog::PluginConfigDialog() {
 	grid->add_child(script_name_label);
 
 	script_edit = memnew(LineEdit);
-	script_edit->set_tooltip_text(TTR("Optional. The path to the script (relative to the add-on folder). If left empty, will default to \"plugin.gd\"."));
+	script_edit->set_tooltip_text(TTR("Optional. The name of the script file. If left empty, will default to the subfolder name."));
```

Comment:
What does "subfolder" mean in this context? Maybe this should be more explicit.

Reply:
It's the name of one of the inputs so I think it means a specific thing in this context and rephrasing it will be more confusing?

![sf](https://github.com/user-attachments/assets/9e40876a-430f-45a7-b023-ab376a2c7d17)

Reply:
Sure, this is fine then.